src/main/java/com/mraof/minestuck/item/StructureScannerItem.java

```diff
@@ -21,65 +37,112 @@ public class StructureScannerItem extends Item
 {
 	private final TagKey<Structure> structure;
 	private final Supplier<Item> fuelItem;
+	private final String scannerName;
 	
-	public StructureScannerItem(Properties properties, TagKey<Structure> structure, Supplier<Item> fuelItem)
+	public StructureScannerItem(Properties properties, TagKey<Structure> structure, Supplier<Item> fuelItem, String scannerName)
```

Comment:
what is the purpose of "scannerName"? If its needed later, it can be acquired via the registry name of the item instance

Reply:
'~' ) I... don't know, honestly. I forget why that's there.

Deleted scannerName.